Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1421,9 +1430,7 @@ - (void)pin_addOperationWithQueuePriority:(PINRemoteImageManagerPriority)priorit
 {
     NSBlockOperation *operation = [NSBlockOperation blockOperationWithBlock:block];
     operation.queuePriority = operationPriorityWithImageManagerPriority(priority);
-    if ([operation respondsToSelector:@selector(qualityOfService)]) {
-        operation.qualityOfService = NSOperationQualityOfServiceBackground;
-    } else {
+    if (supportsQOS == NO) {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
         operation.threadPriority = 0.2;
```

Comment:
Sorta curious where this value came from.  Relevant for iPhone 4 / iOS 7, where this perhaps counts more than on any other device!